src/components/button/button.scss

```diff
@@ -21,6 +21,9 @@ button {
     &.mdc-button {
         min-width: pxToRem(36);
         padding: 0 pxToRem(12);
+        height: 100%;
+        min-height: pxToRem(36);
```

Comment:
I changed this to using `pxToRem`. My bad, since I used plain rem in my comment on the issue 🙂

Reply:
I'm only thinking rem now so my bad not noticing the use of px in the rules next to my additions! 😁